docs/source/contents/4_draft_mapping.rst

```diff
@@ -1662,6 +1662,453 @@ opaque:hostItem
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+
+    <https://example.org/objects/1>
+        relators:rps <http://id.loc.gov/authorities/names/no2017033007> .
+
+physicalLocation as string (UTK)
+--------------------------------
+
+Use Case
+^^^^^^^^
+In many of our own collections, we use strings to describe physicalLocation. We are inconsistent in the structuring of these strings, leading to many different variations for both our Libraries and Special Collections, some of these include misspellings or just different formatting:
+
+- The University of Tennessee Libraries, Knoxville
+- University of Tennesse Knoxville. Libraries
+- University of Tennessee Knoxville. Libraries
+
+Justification
+^^^^^^^^^^^^^
+To create better consistency and cleanliness going forward, we will isolate all instances of these strings and transcribe them to appropriate URIs for UT Libraries and Special collections.
+
+Xpath
+^^^^^
+
+:code:`mods:location/mods:physicalLocation`
+
+Decision
+^^^^^^^^
+Even when mods only has a string present, we will map The University of Tennessee Libraries, Knoxville and University of Tennessee, Knoxville. Special Collections to relative URIs.
+
+`Example record from fbpro:94819 <https://digital.lib.utk.edu/collections/islandora/object/fbpro:94819/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>The University of Tennessee Libraries, Knoxville</physicalLocation>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+
+    <https://example.org/objects/1>
+        relators:rps <http://id.loc.gov/authorities/names/n80003889> .
+
+`Example record from cDanielCartoon:1177 <https://digital.lib.utk.edu/collections/islandora/object/cDanielCartoon:1177/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Tennessee, Knoxville. Special Collections</physicalLocation>
+        <holdingSimple>
+            <copyInformation>
+                <shelfLocator>Taxes and Economy</shelfLocator>
+            </copyInformation>
+        </holdingSimple>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+
+    <https://example.org/objects/1>
+        relators:rps <http://id.loc.gov/authorities/names/no2014027633> .
+
+physicalLocation as string (non-UTK)
+------------------------------------
+
+Use Case
+^^^^^^^^
+Across our collections, there are also many cases where non-UTK items do not have URIs and instead use strings.
+
+Justification
+^^^^^^^^^^^^^
+Translating these to a relative URIs would require significant effort, and the value added may be trivial at this point.
+
+Xpath
+^^^^^
+
+:code:`mods:location/mods:physicalLocation`
```

Comment:
Again, this xpath is super general and is capturing any physicalLocation in a location.  Maybe this would be better:

```
:code:`mods:location/mods:physicalLocation[not(@valueURI)][not(text()="The University of Tennessee Libraries, Knoxville" or text()="University of Tennesse Knoxville. Libraries" or text()="University of Tennessee Knoxville. Libraries")]`
```